layouts/community/ortho_5x12/xyverz/config.h

```diff
@@ -6,8 +6,8 @@
 #define BACKLIGHT_LEVELS 8
 
 /* ws2812 RGB LED */
-#undef RGB_DI_PIN
```

Comment:
Actually, it would be better if it was:

```c
#ifdef RGB_DI_PIN
#undef RGB_DI_PIN
#endif
```

Also, I'm not a fan of editing other people's keymaps, especially if they are pretty active.